GameTests.java

```diff
@@ -1,14 +1,76 @@
 import org.junit.Test;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 
 import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
 public class GameTests {
     @Test
-    public void playerMoveIsRegistered() {
-        Game game = new Game(System.in);
+    public void playerMoveIsRegistered() throws IOException {
+        InputStream stream = new ByteArrayInputStream("2".getBytes());
         Board board = new Board("---------");
-        assertThat(game.playerMakesMove(board, 2).getState(), is("-x-------"));
+        Game game = new Game(stream, board);
+        assertThat(game.playerMakesMove().getState(), is("-o-------"));
+    }
+
+    @Test
+    public void multiplePlayerMovesAreRegistered() throws IOException {
+        InputStream stream = new ByteArrayInputStream("2\n3".getBytes());
+        Board board = new Board("---------");
+        Game game = new Game(stream, board);
+        assertThat(game.playerMakesMove().getState(), is("-o-------"));
```

Comment:
this has already been tested in the test 'playerMoveIsRegistered' so you could just do:
game.playerMakesMove()
assertThat(game.playerMakesMove().getState(), is("_ox------"));